packages/node_modules/express-pouchdb/lib/index.js

```diff
@@ -185,6 +185,38 @@ module.exports = function (startPouchDB, opts) {
     next();
   });
 
+  app.use(function (req, res, next) {
+    var regexFindDbName = /\/([^\/]*)/
+    var dbName = decodeURIComponent(req.url.match(regexFindDbName)[1])
+
+    if (!dbName) {
+      return next();
+    }
+
+    // check for internal routes GET /_all_dbs
+    if (dbName[0] === '_') {
+      for (var i = 0, route; i < app._router.stack.length; i++) {
+        route = app._router.stack[i].route;
+        if (route) for (var j = 0, method; j < route.stack.length; j++) {
+          method = route.stack[j].method.toUpperCase()
+          if (method === req.method && route.path.substr(1, dbName.length) === dbName) {
+            console.log('ok?')
+            return next();
+          }
+        };
+      }
+    }
```

Comment:
Ah actually I see now why you had to do this; you need to send the `400`.

I agree that looping through all routes for every single request seems like a potentially slow design, though. I'd prefer a more elegant solution but you'd probably have to dig through the code to see how Marten set up the routes because I'm not an expert in that.